src/utils/utils.py

```diff
@@ -11,7 +11,7 @@
 import codecs
 import time
 
-from nltk.tokenize.moses import MosesTokenizer, MosesDetokenizer
+from mosestokenizer import MosesTokenizer, MosesDetokenizer
```

Comment:
If this is a separate package from the nltk download, can you add it to the Dockerfile as well?
https://github.com/jsalt18-sentence-repl/jiant/blob/master/Dockerfile